pkg/cli/image_cmd_test.go

```diff
@@ -376,7 +376,20 @@ func TestSignature(t *testing.T) {
 		str := space.ReplaceAllString(buff.String(), " ")
 		actual := strings.TrimSpace(str)
 		So(actual, ShouldContainSubstring, "IMAGE NAME TAG DIGEST SIGNED SIZE")
-		So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 true")
+		So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 true 15B")
+
+		t.Log("Test getting all images using rest calls to get catalog and individual manifests")
```

Comment:
What is the difference between this test and the previous one?

Reply:
I modified it as mentioned in the task description

Reply:
Nevermind. I did not notice the `cmd` value change.